### PR TITLE
fix pyiron table

### PR DESCRIPTION
**Pull request by jan-janssen**

### CODE REVIEW

File: pyiron/table/datamining.py [hunk 1]
```diff
@@ -802,6 +802,7 @@ def always_true_pandas(job_table):
         pandas.Series: A series of True values
 
     """
+    import pandas
```
Comment (sudarsan-surendralal) [inline pyiron/table/datamining.py:805]: Why are you reimporting pandas?

Comment (jan-janssen) [reply pyiron/table/datamining.py:805]: Exactly - when reloading the function from the HDF5 file all import statements have to be included in the same function. This broke my calculations.

Comment (sudarsan-surendralal) [reply pyiron/table/datamining.py:805]: In that case, it's better to import `pandas.Series`  as `Series` in the function. This avoids the unpythonic reimporting of the `pandas` module.

File: pyiron/table/datamining.py [hunk 2]
```diff
@@ -802,7 +802,8 @@ def always_true_pandas(job_table):
         pandas.Series: A series of True values
 
     """
-    return pandas.Series([True] * len(job_table), index=job_table.index)
+    import pandas.Series as Series
```
Comment (sudarsan-surendralal) [inline pyiron/table/datamining.py:805]: Sorry! It's supposed to be `from pandas import Series`

**Pull request merged**